NEWS.md

```diff
@@ -47,6 +47,12 @@ Language changes
   used to yield the string " a\nb", since the single space before `b` set the indent level.
   Now the result is "a\n b", since the space before `b` is no longer considered to occur
   at the start of a line. The old behavior is considered a bug ([#35001]).
+
```

Comment:
Remove the extra line break please; see the formatting in the rest of the file.

Reply:
Sorry! removed in eb6df88